.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,105 @@
+name: Build
+
+on:
+    push:
+        branches:
+            - master
+        tags:
+            - "v*"
+    pull_request:
+        branches:
+            - master
+
+jobs:
+    Build:
+        strategy:
+            matrix:
+                options:
+                    - os: ubuntu-latest
+                      framework: netcoreapp3.1
+                      runtime: -x64
+                      codecov: false
+                    - os: windows-latest
+                      framework: netcoreapp3.1
+                      runtime: -x64
```

Comment:
It's worth to also test x86 + 3.1 because all of the floating point magic we do.